src/components/Images/Images.js

```diff
@@ -199,6 +211,57 @@ export const filePath = (publicId) => {
   return publicId.replace(publicIdBase, '')
 }
 
+/**
+ *  Sourced from here: https://web.dev/preload-responsive-images/#preload-and-lesspicturegreater
+ *
+ *  Output something like the below
+ *  <link rel="preload" href="small_cat.jpg" as="image" media="(max-width: 400px)" />
+ *  <link rel="preload" href="medium_cat.jpg" as="image" media="(min-width: 400.1px) and (max-width: 800px)" />
+ *  <link rel="preload" href="large_cat.jpg" as="image" media="(min-width: 800.1px)" />
+ *
+ *  When Safari supports imagesrcset, this is unnecessary
+ */
+
+export const PreloadTags = ({ crop, publicId, height, width }) => {
```

Comment:
Should the name of this component more closely reflect that it's meant for use with images? `PreloadImageTag`?